src/Uno.UWP/Devices/Sensors/SimpleOrientationSensor.Android.cs

```diff
@@ -107,6 +118,81 @@ private static Orientation DefaultDeviceOrientation
 				}, null);
 		}
 
+		partial void StartListeningOrientationChanged()
+		{
+			var mainLooper = Looper.MainLooper;
+			var context = ContextHelper.Current;
+
+			// Thread pool is used to avoid startup
+			// cost of threads creation.
+			ThreadPool.QueueUserWorkItem(
```

Comment:
What if the user starts and immediately stops? Then the queued work item would execute only after - which would mean the stop would have no effect and the sensor would start running anyway

Reply:
I see. Let me run a few manual tests and validate this. Will keep you posted

Reply:
Validated: The StartListening method is not public and is only available from the event registration. Having said so, the chances of a user starting and stopping listening immediately are very minimal (close to zero)

cc: @MartinZikmund